core/templates/dev/head/css/oppia.css

```diff
@@ -3393,13 +3393,12 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-contributors-more-circle {
-  background: #888;
-  border-radius: 50%;
-  border-color: black;
+  background: grey;
+  color: white;
+  border-radius: 10%;
   height: 20px;
-  padding-top: 3px;
-  text-align: center;
-  width: 20px;
+  width: auto;
+  min-width: 20px;
```

Comment:
Could you please alphabetize the attrs? I.e. background then border-radius then color etc.

Makes it easy to search for attrs when there are a lot of them, and prevents accidental duplication.